config_gen.py

```diff
@@ -401,7 +404,10 @@ def parse_flags(build_log):
     # Only specify one word size (the largest)
     # (Different sizes are used for different files in the linux kernel.)
     mRegex = re.compile("^-m[0-9]+$")
-    word_flags = list([f for f in flags if isinstance(f, basestring) and mRegex.match(f)])
+    if sys.version_info[0] == 2:
+        word_flags = list([f for f in flags if isinstance(f, basestring) and mRegex.match(f)])
+    elif sys.version_info[0] == 3:
+        word_flags = list([f for f in flags if isinstance(f, str) and mRegex.match(f)])
```

Comment:
This would be more concise if you factored out the instance of check for string into a function.